### PR TITLE
Include Vaults in tax report, plus other things

### LINKED ISSUE #1013: Take Vault losses (interest rate + liquidation) into account in the tax report
## Problem Definition

Add a new category, lending and take it into account in the tax report. Also mention it in the changelog for 707 (would only work for premium users)

Should be easy

The only problem is that exact loss in a given time range is going to be hard to calculate. Loss will only be realized at full payback of a loan.

### PR DESCRIPTION
**Pull request by LefterisJP**
Fix #1013 

Also 

- [X] Adds vault owners and shows them in the UI
- [x] Adds stability fee and shows it in the UI
- [x] Rename vault name to collateral type

### CODE REVIEW

File: electron-app/src/services/types-api.ts [hunk 1]
```diff
@@ -57,6 +57,7 @@ export interface ApiManualBalances {
 export interface ApiMakerDAOVault {
   readonly identifier: number;
   readonly name: string;
```
Comment (isidorosp) [inline electron-app/src/services/types-api.ts:59]: Shall we change `name` to `collateral_type` to be in line with the maker docs?
image
https://github.com/makerdao/dss/wiki/Glossary

Comment (LefterisJP) [reply electron-app/src/services/types-api.ts:59]: It's in the todos of this PR.

Comment (LefterisJP) [reply electron-app/src/services/types-api.ts]: done.

**Comment by LefterisJP** [comment]
@isidorosp I will merge this. I am not really using the new stuff I added in the frontend but I think you can use them to make it look nicer as you showed in discord. Is that okay?

**Comment by isidorosp** [comment]
> @isidorosp I will merge this. I am not really using the new stuff I added in the frontend but I think you can use them to make it look nicer as you showed in discord. Is that okay?

Yep sounds good!

**Pull request merged**